dream-big-api/app/api/avatar_heads_api.rb

```diff
@@ -39,10 +39,10 @@ class AvatarHeadsApi < Grape::API
 
     # Auth
 
-    result = AvatarHead.find(params[:id])
-    result.update! avatar_heads_parameters
+    updateHead = AvatarHead.find(params[:id])

```

Comment:
Instead of omitting avatar is `updateHead` can we keep it like `updated_avatar_head`